src/abstractdatatable/io.jl

```diff
@@ -1,274 +0,0 @@
-    printtable(STDOUT,
-               dt,
-               header = header,
-               separator = separator,
-               quotemark = quotemark,
-               nastring = nastring)
-    return
-end
-
-"""
-Write data to a tabular-file format (CSV, TSV, ...)
-
-```julia
-writetable(filename, dt, [keyword options])
-```
-
-### Arguments
-
-* `filename::AbstractString` : the filename to be created
-* `dt::AbstractDataTable` : the AbstractDataTable to be written
-
-### Keyword Arguments
-
-* `separator::Char` -- The separator character that you would like to use. Defaults to the output of `getseparator(filename)`, which uses commas for files that end in `.csv`, tabs for files that end in `.tsv` and a single space for files that end in `.wsv`.
-* `quotemark::Char` -- The character used to delimit string fields. Defaults to `'"'`.
-* `header::Bool` -- Should the file contain a header that specifies the column names from `dt`. Defaults to `true`.
-* `nastring::AbstractString` -- What to write in place of missing data. Defaults to `"NULL"`.
-
-### Result
-
-* `::DataTable`
-
-### Examples
-
-```julia
-dt = DataTable(A = 1:10)
-writetable("output.csv", dt)
-writetable("output.dat", dt, separator = ',', header = false)
-writetable("output.dat", dt, quotemark = '\', separator = ',')
-writetable("output.dat", dt, header = false)
-```
-"""
-function writetable(filename::AbstractString,
-                    dt::AbstractDataTable;
-                    header::Bool = true,
-                    separator::Char = getseparator(filename),
-                    quotemark::Char = '"',
-                    nastring::AbstractString = "NULL",
-                    append::Bool = false)
-
-    if endswith(filename, ".bz") || endswith(filename, ".bz2")
-        throw(ArgumentError("BZip2 compression not yet implemented"))
-    end
-
-    if append && isfile(filename) && filesize(filename) > 0
-        file_dt = readtable(filename, header = false, nrows = 1)
-
-        # Check if number of columns matches
-        if size(file_dt, 2) != size(dt, 2)
-            throw(DimensionMismatch("Number of columns differ between file and DataTable"))
-        end
-
-        # When 'append'-ing to a nonempty file,
-        # 'header' triggers a check for matching colnames
-        if header
-            if any(i -> @compat(Symbol(file_dt[1, i])) != index(dt)[i], 1:size(dt, 2))
-                throw(KeyError("Column names don't match names in file"))
-            end
-
-            header = false
-        end
-    end
-
-    openfunc = endswith(filename, ".gz") ? gzopen : open
-
-    openfunc(filename, append ? "a" : "w") do io
-        printtable(io,
-                   dt,
-                   header = header,
-                   separator = separator,
-                   quotemark = quotemark,
-                   nastring = nastring)
-    end
-
-    return
-end
-
-##############################################################################
```

Comment:
Please keep HTML and LaTeX output, these are completely different features that have no replacement in CSV.jl.